portfolio/src/main/webapp/comments.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="description" content="Olu's Portfolio">
+    <meta name="keywords" content="HTML, CSS, JavaScript, Olu, Portfolio, CS">
+    <meta name="author" content="Olu Alamutu">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Olu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <link rel="apple-touch-icon" sizes="180x180" href="images/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="images/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="images/favicon-16x16.png">
+    <link rel="manifest" href="images/site.webmanifest">
+  </head>
+  <body>
+    <div class="row">
+      <img src="images/logo.png"
+    </div>   
+    <ul class="main-nav">    
+      <li><a href="index.html" style="color:white"> HOME </a></li>
+      <li><a href="gallery.html" style="color:white"> PHOTOGRAPHY </a> </li>
+	  <li><a href="showcase.html" style="color:white"> PROJECT SHOWCASE </a></li>
+      <li><a href="about.html" style="color:white"> ABOUT </a></li>
+      <li><a href="contact.html" style="color:white"> CONTACT </a></li>
+      <li><a href="stock.html" style="color:white"> STOCK PICKER</a></li>
+      <li class="active"><a href="comments.html" style="color:white"> COMMENTS</a></li>
+    </ul>
+  </body>
+  <body>
```

Comment:
Can we remove the second `<body>` tag and it's corresponding closing tag? Typically HTML documents have only one `<body>` tag.

Reply:
Addressed